packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -313,6 +315,65 @@ class _CupertinoPickerState extends State<CupertinoPicker> {
   }
 }
 
+/// A default selection overlay for [CupertinoPicker]s.
+///
+/// It draws a gray rounded rectangle to match the picker visuals introduced in iOS 14
+class CupertinoPickerDefaultSelectionOverlay extends StatelessWidget {
+
+  /// Create a selection overlay.
+  ///
+  /// The [background] argument default value is [CupertinoColors.tertiarySystemFill].
+  /// It must be non-null.
+  ///
+  /// The [capLeftEdge] and [capRightEdge] arguments decides whether there is a default margin and radius on the left and right sides.
+  /// They default value is true.
+  /// They must be non-null.
+  const CupertinoPickerDefaultSelectionOverlay({
+    Key key,
+    this.background = CupertinoColors.tertiarySystemFill,
+    this.capLeftEdge = true,
+    this.capRightEdge = true,
+  }) : assert(background != null),
+       assert(capLeftEdge != null),
+       assert(capRightEdge != null),
+       super(key: key);
+
+  /// Whether to use the default radius and margin on the left side
+  final bool capLeftEdge;
+
+  /// Whether to use the default radius and margin on the right side
+  final bool capRightEdge;
+
+  /// The color to fill in the background of the [CupertinoPickerDefaultSelectionOverlay].
+  /// It Support for use [CupertinoDynamicColor].
```

Comment:
I think that's the default for cupertino widgets so no need to state that (unless this is an exception).

Reply:
nit: 
maybe add
```
Typically this should not be set to a fully opaque color, as the currently selected item of the underlying [CupertinoPicker] should remain visible.

Defaults to [CupertinoColors.tertiarySystemFill]. 
```

Reply:
Thank you very much, I have never written so many documents in English before, thank you very much for helping me.

The background default document already exists in the construction method, so I don't see the need to duplicate it.

Reply:
Generally the field itself the place to document the specifics, by convention we should probably still document the default value there. I think having duplication like that is fine, as it's one short sentence and contains vital information.
on a side note, could you update the line breaks of the docs to <= 80 characters, so they conform to the style guide (https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-a-maximum-line-length-of-80-characters)?

Reply:
It's been updated now.